src/reconcile/rec_visibility.c

```diff
@@ -62,6 +62,14 @@ __rec_append_orig_value(
         if (WT_UPDATE_DATA_VALUE(upd) && __wt_txn_upd_visible_all(session, upd))
             return (0);
 
+        /*
+         * Don't add the original value if there is an update restored from history store for
+         * rollback to stable operation. Another way of indicating that we don't need on-disk value
+         * anymore.
```

Comment:
The second sentence is not a complete sentence, just a fragment. I suggest something like:
If the update is restored from the history store for the rollback to stable operation we don't need the on-disk value anymore and we're done.

Reply:
Fixed.